lib/foreman_remote_execution/engine.rb

```diff
@@ -102,8 +102,8 @@ class Engine < ::Rails::Engine
         ]
 
         # Add a new role called 'Remote Execution User ' if it doesn't exist
-        role 'Remote Execution User', USER_PERMISSIONS
-        role 'Remote Execution Manager', MANAGER_PERMISSIONS
+        role 'Remote Execution User', USER_PERMISSIONS, 'TODO'
+        role 'Remote Execution Manager', MANAGER_PERMISSIONS, 'TODO'
```

Comment:
Gives the user permissions to manage job templates, manage remote execution features, cancel jobs and view audit logs.